tests/integration/key_assignment.go

```diff
@@ -81,6 +99,7 @@ func (s *CCVTestSuite) TestKeyAssignment() {
 		},
 		{
 			"double same-key assignment in same block by different vals", func(pk *providerkeeper.Keeper) error {
+				// establish CCV channel
```

Comment:
this comment is extra, its already in the line below

Reply:
I removed the additional comment.